spec/graphql/queries/sightings/sighting_by_location_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'rails_helper'
+
+module Queries
+  RSpec.describe SightingByLocation, type: :request do
```

Comment:
This tests look good, but I'm wondering if it would also be good to have "Request" level tests. For example, a test where you send a `post` request to `/graphql` with the required graphql query string in the body. That would more clearly illustrate what would be expected from the Frontend.